mayastor/src/lvm/error.rs

```diff
@@ -0,0 +1,16 @@
+use snafu::{Snafu};
+
+#[derive(Debug, Snafu)]
+#[snafu(visibility = "pub(crate)")]
+pub enum Error {
+    #[snafu(display("Failed to parse {}", err))]
+    FailedParsing { err: String },
+    #[snafu(display("Failed to execute command {}", err))]
+    FailedExec { err: String },
+    #[snafu(display("{} Not found", err))]
+    NotFound { err: String},
```

Comment:
Already covered by [`std::io::ErrorKind::NotFound`](https://doc.rust-lang.org/nightly/std/io/enum.ErrorKind.html#variant.NotFound). We recently discussed how we should stop adding more error types and variants so we can start here. :) 

```suggestion
    #[snafu(display("I/O error: {}", err))]
    Io { err: std::io::Error },
```

This way, in the future you could emit any of the errors already covered by `std::io::Error` w/o adding new variants. It'll also make it easier to unify/merge error types (where needed) in the future.